docassemble/ALWeaver/data/sources/test_1.feature

```diff
@@ -8,8 +8,10 @@ Notes:
 Scenario: I weave the civil docketing statement
   Given I start the interview at "assembly_line.yml"
   And the max seconds for each Step is 90
+  Then I tap the "#upload" element and wait 5 seconds
```

Comment:
I made a commit that that changed this already, but it needed `wait 5 seconds`, not `wait for 5 seconds`. I think the docs (that I wrote 🙈 ) were wrong there, though I like the `wait for` version as well, so I'll patch that soon.

Waiting to see if the commit I made fixes it.